src/app/app.component.ts

```diff
@@ -1,10 +1,15 @@
-import { Component } from '@angular/core';
+import { Component, ElementRef,ViewChild } from '@angular/core';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
+
 export class AppComponent {
   title = 'project-heart';
+
+  navbarToggler(el): void {
+     el.classList.toggle('active')
+  }
```

Comment:
merko yaad aya ki tu method puch rha tha class toggle krne k lie yaha pr to hum `[ngClass]` ka use krte angular mai
like jo span mai tune click listener call kia uski jgh `navbarToggler` uski  jgh pr
```
[ngClass] = "{'active': method_name(), 'not_active': !method_name()}"
```
.To yaha pr active and not_acitve will be some classes of some html tags and also in css and this method_name will return a boolean value to either set that to class to true or not